client/memez/projects/memez/src/app/views/layout/layout.component.ts

```diff
@@ -1,17 +1,18 @@
 import {Component, OnInit} from '@angular/core';
-import {Router} from '@angular/router';
-import {RootStore} from '../../stores/root.store';
-import {PostStore} from '../../stores/entities/post.store';
-import {UserStore} from '../../stores/entities/user.store';
-import {LoginStore} from '../../stores/views/login.store';
-import {LikeStore} from '../../stores/entities/like.store';
-import {FeedStore} from '../../stores/views/feed.store';
-import {LayoutStore} from "../../stores/views/layout.store";
+import {Router}            from '@angular/router';
+import {RootStore}         from '../../stores/root.store';
+import {PostStore}         from '../../stores/entities/post.store';
+import {UserStore}         from '../../stores/entities/user.store';
+import {LoginStore}        from '../../stores/views/login.store';
+import {LikeStore}         from '../../stores/entities/like.store';
+import {FeedStore}         from '../../stores/views/feed.store';
+import {LayoutStore}       from "../../stores/views/layout.store";
+import {SignupStore}       from "../../stores/views/signup-store.service";
 
 @Component({
```

Comment:
for view components (not reusable components) and this goes for all your view components as well you should use Change Detection strategy onPush for MUCH better performance